src/reducers/index.js

```diff
@@ -72,7 +72,7 @@ function getInitialState() {
 
     /*
      * Output state passed as props to the the rendered FixedDataTable
-     * NOTE (jordan) rows may contain undefineds if we don't need all the buffer positions
+     * NOTE (jordan) rowsToRender may contain undefined(s) if we don't need all the buffer positions
```

Comment:
changes for tests incoming

most of them are because:
* `rows` renamed to `rowsToRender` 
* `columnAnchor` had failed some asserts